rpm.mk

```diff
@@ -82,6 +88,14 @@ local-archive: build-cockpit
 	-mkdir -p dist/$(NAME_VERSION)
 	rsync -a --exclude=node_modules --exclude=dist --exclude=__pycache__ --exclude=.git --exclude=rpmbuild . dist/$(NAME_VERSION)
 
+prep_bundle_libdb:
+	/bin/rm -rf dist/sources/libdb_bundle
+	mkdir -p dist/sources/libdb_bundle/SOURCES rpmbuild/SOURCES
+	cd dist/sources/libdb_bundle && curl -LO $(LIBDB_URL)
```

Comment:
This approach won't work in koji, because there is no internet access.

Reply:
I am puzzled. How jemalloc is handled then ? 
    if [ $(BUNDLE_JEMALLOC) -eq 1 ]; then \
        curl -LO $(JEMALLOC_URL) ; \
    fi ;

Reply:
For offline builds sources are first uploaded to lookaside cache. For example jemalloc tarballs are located for our package in Fedora under https://src.fedoraproject.org/repo/pkgs/389-ds-base/. We can create a tarball with libdb+patches and upload it there as well.